$MirOS: ports/mailnews/pine/patches/patch-doc_pilot_1,v 1.1.7.1 2005/03/18 15:49:48 tg Exp $
--- doc/pilot.1.orig	Fri Jan 14 20:39:58 2005
+++ doc/pilot.1	Wed May  4 23:28:07 2005
@@ -1,4 +1,4 @@
-.TH pilot 1 "Version 1.1"
+.TH pilot 1 "Version 2.0L"
 .SH Name
 pilot \- simple file system browser in the style of the Pine Composer
 .SH Syntax
@@ -9,6 +9,9 @@ pilot \- simple file system browser in t
 .I directory
 ]
 .SH Description
+Attention: this manual page describes \fIpilot\fR 1.1 instead of the
+actual compiled 2.0 version.
+.PP
 \fIPilot\fR is a simple, display-oriented file system browser based on
 the Pine message system composer.  As with Pine, commands are 
 displayed at the bottom of the screen, and context-sensitive
